Alicerce/Sources/Logging/Log+FileLogDestination.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  Log+FileLogDestination.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class FileLogDestination: LogDestination {
+
+        private static let dispatchQueueLabel = "com.mindera.Alicerce.FileLogDestination"
+
+        public private(set) var dispatchQueue: DispatchQueue
+        public private(set) var minLevel: Level
+        public private(set) var formatter: LogItemFormatter
+        public var instanceId: String {
+            return "\(type(of: self))_\(fileURL.absoluteString)"
+        }
+
+        internal let fileURL: URL
+        internal let fileManager = FileManager.default
+
+        //MARK:- lifecycle
+
+        public init(fileURL: URL,
+                    minLevel: Level = Log.Level.error,
+                    formatter: LogItemFormatter = Log.StringLogItemFormatter(),
+                    dispatchQueue: DispatchQueue = DispatchQueue(label: FileLogDestination.dispatchQueueLabel)) {
+            
+            self.fileURL = fileURL
+            self.minLevel = minLevel
+            self.formatter = formatter
+            self.dispatchQueue = dispatchQueue
+        }
+
+        //MARK:- public Methods
+
+        public func clear() {
+            guard fileManager.fileExists(atPath: fileURL.path) else { return }
+            do {
+                try fileManager.removeItem(at: fileURL)
+            }
+            catch {
```

Comment:
Well, I changed it a little so a completion closure is being called with the outcome. The default handler is being reused for all destinations and logs to the console if something went wrong.